docs/source/contributors.rst

```diff
@@ -9,14 +9,28 @@ Contributors
    Solarflare
 
 cocotb is developed and maintained by an active community.
-Our GitHub repository contains a list of the :reposrc:`Maintainers <CONTRIBUTING.md#maintainers>` and all
-`contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_.
+Our GitHub repository contains a list of the `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_.
 
 Since mid-2018 cocotb is an independent project under the umbrella of the
-`Free and Open Source Silicon Foundation <https://www.fossi-foundation.org>`_.
+`FOSSi Foundation <https://www.fossi-foundation.org>`_.
 The FOSSi Foundation provides the cocotb project with financial,
 legal and administrative support, and holds cocotb assets.
 
+.. _maintainers:
+
+Maintainers
+===========
+
+Cocotb uses a shared maintainer model.
+Most maintainers are experts in part of the cocotb codebase, and are primarily responsible for reviews in this area.
+
+- Kaleb Barrett (@ktbarrett)
+- Tomasz Hemperek (@themperek)
+- Marlon James (@marlonjames)
+- Colin Marquardt (@cmarqu)
+- Philipp Wagner (@imphil)
+- Eric Wieser (@eric-wieser)
```

Comment:
I removed the "Founders" list here since it's duplicated with the names just below this paragraph.